salt/proxy/cimc.py

```diff
@@ -97,6 +97,16 @@ def __virtual__():
     return __virtualname__
 
 
+def _validate_response_code(response_code_to_check, cookie_to_logout=None):
+    formatted_response_code = str(response_code_to_check)
+    if formatted_response_code not in ['200', '201', '202', '204']:
+        if cookie_to_logout:
+            logout(cookie_to_logout)
+        log.debug("Received error HTTP status code: {0}" .format(formatted_response_code))
```

Comment:
Perhaps `log.error` would be more appropriate?

Reply:
@spenceation What do you think about this comment here?